docs/en/advanced_guides/models.md

```diff
@@ -1 +1,119 @@
 # Models
+
+We usually define a neural network in a deep learning task as a model, and this model is the core of an algorithm. [MMEngine](https://github.com/open-mmlab/mmengine) abstracts a unified model [BaseModel](https://github.com/open-mmlab/mmengine/blob/main/mmengine/model/base_model/base_model.py#L16) to standardize the interfaces for training, testing and other processes. All models implemented by MMSegmentation inherit from `BaseModel`, but we have re-implemented some of the interfaces for semantic segmentation tasks.
```

Comment:
but we have re-implemented some of the interfaces for semantic segmentation tasks. ->

and in mmseg we implemented `forward` and add some functions for semantic segmentation algorithm